Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerSwizzlingTests.swift

```diff
@@ -11,30 +11,39 @@ class SentryUIViewControllerSwizzlingTests: XCTestCase {
         let objcRuntimeWrapper = SentryTestObjCRuntimeWrapper()
         let subClassFinder: TestSubClassFinder
         let processInfoWrapper = SentryNSProcessInfoWrapper()
+        let binaryImageCache: SentryBinaryImageCache
         
         init() {
             subClassFinder = TestSubClassFinder(dispatchQueue: dispatchQueue, objcRuntimeWrapper: objcRuntimeWrapper)
+            binaryImageCache = SentryDependencyContainer.sharedInstance().binaryImageCache
```

Comment:
What's the point of doing this? Why add another parameter to SentryUIViewControllerSwizzling where it was already using the singleton from SentryDependencyContainer, just to assign it to the same value again in these tests? It's not even accessed anywhere in the tests.

Reply:
Following @philipphofmann suggestion to use injection in the constructor instead of making direct use of `SentryDependencyContainer`

Reply:
Yep, that's following our decision on how to handle [dependency injection](https://github.com/getsentry/sentry-cocoa/blob/main/develop-docs/DECISIONS.md#dependency-injection-strategy). I'm a fan of specifying all dependencies via their constructor to improve testability. If you disagree, I'm happy to discuss this again, but not here in this PR 😃

Reply:
i mean, we're not injecting anything, we just added it and aren't using it. yagni, do this if/when we actually need to inject a mock